### PR TITLE
Diff filepicker

### PR DESCRIPTION
**Pull request by ubitux**

### CODE REVIEW

File: pynopegl-utils/pynopegl_utils/diff/__init__.py [hunk 1]
```diff
@@ -40,23 +40,23 @@ def __init__(self, qml_engine, ngl_widget, args):
...
-        self._player.timeChanged.connect(ngl_widget.set_time)
-        self._player.mouseDown.connect(self._mouse_down)
-        self._player.zoom.connect(self._zoom)
-        self._player.pan.connect(self._pan)
+        player = app_window.findChild(QObject, "player")
+        player.timeChanged.connect(ngl_widget.set_time)
+        player.mouseDown.connect(self._mouse_down)
+        player.zoom.connect(self._zoom)
+        player.pan.connect(self._pan)
+        self._player = player
```
Comment (mbouron) [inline pynopegl-utils/pynopegl_utils/diff/__init__.py:58]: Are we sure that the event can't be fired before self._player is properly set ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 - `%s_coord_matrix` uniform is now exposed for 2D array and 3D textures
 - `ngl_get_backend()` function to retrieve the backend information associated
   with a configured `nope.gl` context
+- `ngl-diff` can now be run without specifying the input files, which can be set
```
Comment (mbouron) [inline CHANGELOG.md:16]: `ngl-diff` can now set and change the input files from the GUI. While still supported, passing them through the command line is not mandatory anymore.

**Review by mbouron — APPROVED** [review]
Tested locally. Except from my comments, LGTM.

File: pynopegl-utils/pynopegl_utils/diff/__init__.py (hunk 1 above)
Comment (ubitux) [reply pynopegl-utils/pynopegl_utils/diff/__init__.py:58]: Good point. Fortunately the `Diff.__init__()` must be terminated before we start the app:
```python
    diff = _Diff(engine, ngl_widget, sys.argv)
    ret = app.exec()
```

Comment (mbouron) [reply pynopegl-utils/pynopegl_utils/diff/__init__.py:58]: Ok, nice clarification ! Thanks.

**Pull request merged**